webpack.config.js

```diff
@@ -18,9 +18,9 @@ const config = [{
     './modules/configuration/js/SubprojectRelations.js': './modules/configuration/jsx/SubprojectRelations.js',
     './modules/bvl_feedback/js/react.behavioural_feedback_panel.js': './modules/bvl_feedback/jsx/react.behavioural_feedback_panel.js',
     './modules/data_team_helper/js/behavioural_qc_module.js': './modules/data_team_helper/jsx/behavioural_qc_module.js',
-    './modules/candidate_list/js/AccessProfilePanel.js': './modules/candidate_list/jsx/AccessProfilePanel.js',
-    './modules/candidate_list/js/columnFormatter.js': './modules/candidate_list/jsx/columnFormatter.js',
+    './modules/candidate_list/js/AccessProfilePanel.js': './modules/candidate_list/jsx/openProfileForm.js',
```

Comment:
Should this be written `'./modules/candidate_list/js/openProfileForm.js': './modules/candidate_list/jsx/openProfileForm.js',` instead?

Reply:
Oupsie... This is fixed now. Nice catch!